comparadise-utils/match-screenshot.ts

```diff
@@ -115,13 +121,18 @@ export function matchScreenshot(
           'log',
           `❌ Actual image of ${name} differed by ${diffPixels} pixels.`
         );
+        visualDiffsExist = true;
```

Comment:
Why not exit right here?

Reply:
i guess i was thinking it was iterating over something, but that's not the case